README.md

```diff
@@ -127,11 +126,14 @@ Sugarizer Server Dashboard is an admin tool for teachers and deployment administ
 * Graphic and request: display graphics and report on previous data.
 
 To login to the Dashboard the first time, you will have to create an admin account using this command:
-
+        
+        docker exec -it <id of sugarizer-server_server> bash
+	
 	sh add-admin.sh admin password http://127.0.0.1:8080/auth/signup

```

Comment:
If the script is run inside the docker container, the port becomes 80

Reply:
please can you review again